### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by anilnishad19799** [OpenMined/SyferText#201]
I want to secure my nlp model while deploying in on-device , that i have build in spacy.

installation steps that i follows :
git clone https://github.com/OpenMined/SyferText.git
cd SyferText
python setup.py install

## System Information
 - OS: Windows

## Error
```
PS E:\SyferText\SyferText> python setup.py install
C:\Users\Divy\AppData\Local\Programs\Python\Python38\lib\distutils\dist.py:274: UserWarning: Unknown distribution option: 'classifier'
  warnings.warn(msg)
running install
running bdist_egg
running egg_info
writing syfertext.egg-info\PKG-INFO
writing dependency_links to syfertext.egg-info\dependency_links.txt
writing requirements to syfertext.egg-info\requires.txt
writing top-level names to syfertext.egg-info\top_level.txt        
reading manifest file 'syfertext.egg-info\SOURCES.txt'
writing manifest file 'syfertext.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg  
running install_lib
running build_py
creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\syfertext
copying build\lib\syfertext\attrs.py -> build\bdist.win-amd64\egg\syfertext
copying build\lib\syfertext\char_classes.py -> build\bdist.win-amd64\egg\syfertext
creating build\bdist.win-amd64\egg\syfertext\encoders
copying build\lib\syfertext\encoders\default_encoder.py -> build\bdist.win-amd64\egg\syfertext\encoders      
copying build\lib\syfertext\encoders\sentence_encoder.py -> build\bdist.win-amd64\egg\syfertext\encoders    
copying build\lib\syfertext\encoders\__init__.py -> build\bdist.win-amd64\egg\syfertext\encoders
copying build\lib\syfertext\lexeme.py -> build\bdist.win-amd64\egg\syfertext
copying build\lib\syfertext\lex_attrs.py -> build\bdist.win-amd64\egg\syfertext
copying build\lib\syfertext\stop_words.py -> build\bdist.win-amd64\egg\syfertext
creating build\bdist.win-amd64\egg\syfertext\tokenizers
copying build\lib\syfertext\tokenizers\char_classes.py -> build\bdist.win-amd64\egg\syfertext\tokenizers    
copying build\lib\syfertext\tokenizers\default_tokenizer.py -> build\bdist.win-amd64\egg\syfertext\tokenizers
copying build\lib\syfertext\tokenizers\punctuations.py -> build\bdist.win-amd64\egg\syfertext\tokenizers    
copying build\lib\syfertext\tokenizers\spacy_tokenizer.py -> build\bdist.win-amd64\egg\syfertext\tokenizers  
copying build\lib\syfertext\tokenizers\token_exception.py -> build\bdist.win-amd64\egg\syfertext\tokenizers  
copying build\lib\syfertext\tokenizers\utils.py -> build\bdist.win-amd64\egg\syfertext\tokenizers  
copying build\lib\syfertext\tokenizers\__init__.py -> build\bdist.win-amd64\egg\syfertext\tokenizers
creating build\bdist.win-amd64\egg\syfertext\trainers
copying build\lib\syfertext\trainers\__init__.py -> build\bdist.win-amd64\egg\syfertext\trainers    
copying build\lib\syfertext\vocab.py -> build\bdist.win-amd64\egg\syfertext
copying build\lib\syfertext\__init__.py -> build\bdist.win-amd64\egg\syfertext
byte-compiling build\bdist.win-amd64\egg\syfertext\attrs.py to attrs.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\char_classes.py to char_classes.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\encoders\default_encoder.py to default_encoder.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\encoders\sentence_encoder.py to sentence_encoder.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\encoders\__init__.py to __init__.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\lexeme.py to lexeme.cpython-38.pyc      
byte-compiling build\bdist.win-amd64\egg\syfertext\lex_attrs.py to lex_attrs.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\stop_words.py to stop_words.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\char_classes.py to char_classes.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\default_tokenizer.py to default_tokenizer.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\punctuations.py to punctuations.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\spacy_tokenizer.py to spacy_tokenizer.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\token_exception.py to token_exception.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\utils.py to utils.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\tokenizers\__init__.py to __init__.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\trainers\__init__.py to __init__.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\vocab.py to vocab.cpython-38.pyc
byte-compiling build\bdist.win-amd64\egg\syfertext\__init__.py to __init__.cpython-38.pyc
creating build\bdist.win-amd64\egg\EGG-INFO
copying syfertext.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying syfertext.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying syfertext.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying syfertext.egg-info\requires.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying syfertext.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist\syfertext-0.2.0-py3.8.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing syfertext-0.2.0-py3.8.egg
Removing c:\users\divy\appdata\local\programs\python\python38\lib\site-packages\syfertext-0.2.0-py3.8.egg
Copying syfertext-0.2.0-py3.8.egg to c:\users\divy\appdata\local\programs\python\python38\lib\site-packages
syfertext 0.2.0 is already the active version in easy-install.pth

Installed c:\users\divy\appdata\local\programs\python\python38\lib\site-packages\syfertext-0.2.0-py3.8.egg
Processing dependencies for syfertext==0.2.0
Searching for syft==0.4.0
Reading https://pypi.org/simple/syft/
No local packages or working download links found for syft==0.4.0
error: Could not find suitable distribution for Requirement.parse('syft==0.4.0')
```

kindly give me suggestion to protect my spacy model on-device deployment , and please resolve my error.

**Comment by karynaur** [comment]
There was a small issue with the syft version in setup.py. I have made the necessary changes(form syft==0.4.0 to 0.3.0) and it works fine on colab. PR #202

**Comment by Bhuvan-21** [comment]
Thanks a lot, I faced the same issue it works with the changes you have made here.

**Comment by AlanAboudib** [comment]
solved in #211
